components/modals/ProfilePreview.js

```diff
@@ -1,18 +1,42 @@
 import UserPage from "@components/user/UserPage";
+import { useEffect, useRef } from "react";
 
 export default function Preview({ toggle, data }) {
   const BASE_URL = process.env.NEXT_PUBLIC_BASE_URL;
+  const modalContentRef = useRef(null)
+
+

```

Comment:
Thank you for the contribution, I don't think 2 blank lines are needed here

Reply:
Noted